mesa_frames/concrete/agentset.py

```
@@ -86,12 +85,12 @@ class AgentSetPolars(AgentSetDF, PolarsMixin):
     _copy_only_reference: list[str] = ["_model", "_mask"]
     _mask: pl.Expr | pl.Series
 
-    def __init__(self, model: "ModelDF") -> None:
+    def __init__(self, model: mesa_frames.concrete.model.ModelDF) -> None:
```

Ruff (0.8.2)

Undefined name `mesa_frames`

(F821)